app/routes/__index/domains/index.tsx

```diff
@@ -15,20 +20,51 @@ export const loader = async ({ request }: LoaderArgs) => {
   return typedjson(await getRecordsByUsername(username));
 };
 
+export const action = async ({ request }: ActionArgs) => {
+  const { id } = Object.fromEntries(await request.formData());
+  const formalizedID = Number(id);
+
+  if (request.method === 'PUT') {
+    const record = await getRecordById(formalizedID);
+    if (record) {
+      const newExpiresAt = new Date(record.expiresAt);
+      newExpiresAt.setMonth(newExpiresAt.getMonth() + 6);
+
+      await updateRecordById(
```

Comment:
This signature is extremely hard to use and maintain. Please pass in an object instead. You can check the certificate and record models for an example if you would like to